src/granite/querying.cr

```diff
@@ -86,7 +87,7 @@ module Granite::Querying
     end
 
     loop do
-      results = all "#{clause} LIMIT ? OFFSET ?", params + [limit, offset]
+      results = all "#{clause} LIMIT ? OFFSET ?", params + [limit, offset], false
```

Comment:
This is another spot where it should be `true` if this is a model that had a `write` action performed within the set period of time.

Reply:
See previous response.